Mathlib/FieldTheory/Finite/GaloisField.lean

```diff
@@ -222,7 +223,45 @@ end GaloisField
 
 namespace FiniteField
 
-variable {K : Type*} [Field K] [Fintype K] {K' : Type*} [Field K'] [Fintype K']
+variable {K K' : Type*} [Field K] [Field K']
+
+section norm
+
+variable [Algebra K K'] [Finite K']
+
+theorem algebraMap_norm_eq_pow {x : K'} :
+    algebraMap K K' (Algebra.norm K x) = x ^ ((Nat.card K' - 1) / (Nat.card K - 1)) := by
+  have := Finite.of_injective_finite_range (algebraMap K K').injective
```

Comment:
Why do you need `..._finite_range`, not just `Finite.of_injective`?

Reply:
```suggestion
  have := Finite.of_injective (algebraMap K K').injective
```